lib/internal/structured_clone.js

```diff
@@ -20,6 +21,8 @@ function structuredClone(value, options = undefined) {
   channel ??= new MessageChannel();
   channel.port1.unref();
   channel.port2.unref();
+  if (typeof value === 'object' && !ArrayIsArray(value))
+    value = { ...value };
```

Comment:
This won't work for special objects that are transferable, such as error objects